gateway-openapi-ui/src/main/resources/swagger/index_admin.html

```diff
@@ -0,0 +1,39 @@
+<!-- HTML for static distribution bundle build -->
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>The Apache Knox REST API</title>
+    <link rel="stylesheet" type="text/css" href="./swagger-ui.css" >
+    <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
+  </head>
+
+  <body>
+    <div id="swagger-ui"></div>
+
+    <script src="./swagger-ui-bundle.js" charset="UTF-8"> </script>
+    <script src="./swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
+    <script>
+    window.onload = function() {
+      // Begin Swagger UI call region
+      const ui = SwaggerUIBundle({
+        url: "https://localhost:8443/gateway/homepage/home/swagger/admin.json",
```

Comment:
Is localhost always correct here?

Reply:
It should work in most cases, but let me fix it. Thanks!

Reply:
Fixed.